MainForm.cs

```diff
@@ -0,0 +1,173 @@
+﻿using EnumColors;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Task1_GUI
+{
+    public partial class MainForm : Form
+    {
+        public MainForm()
+        {
+            InitializeComponent();
+        }
+
+        bool CreateCardCombination(int combLen, List<Card> cards, TextBox textBox)
+        {
+            bool isCombinationCorrect = true;
+            for (int i = 0; i < combLen; i++)
+            {
+                Card card = new Card(textBox.Lines[i + 1]);
+                isCombinationCorrect = Card.IsCardCorrect;
+                if (!isCombinationCorrect)
+                {
+                    textBox.Text = Service.ErrorMessage;
+                    textBox.ForeColor = Color.Red;
+                    break;
+                }
+                else
+                {
+                    cards.Add(card);
+                }
+            }
+            return isCombinationCorrect;
+        }
+        
+        public void MainFunc(int combLen1, int combLen2)
+        {
+            Label[] labelsOnCards1 = { c11, c12, c13, c14, c15, c16, c17 };
```

Comment:
It's not an error, it's just a note. Declaring all variables at the beginning of the method is old style. Better will be declare variable when you start using it. It will allow me to not scroll up to find variable type every time then I will see some new variable.